utils/src/main/scala/com.gu.contentatom.renderer/utils/CapiService.scala

```diff
@@ -42,7 +42,7 @@ object Main extends IOApp {
     commonsdivisionConfiguration = ArticleConfiguration.CommonsdivisionConfiguration(true)
   )
 
-  val client = new GuardianContentClient(sys.env("CAPI_TEST_KEY"))
+  val client = GuardianContentClient(sys.env("CAPI_TEST_KEY"))
```

Comment:
Relates to this change: https://github.com/guardian/content-api-scala-client/commit/4e2676ca4722e268f9832b5b9d5972fc04902626#diff-66a458e6ed4e61011cc7e9c9d0d3be2a